src/components/Content/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoItem: React.FC<Props> = ({ todo }) => (
+  <div
+    data-cy="Todo"
+    className={classNames(
+      'todo',
+      { completed: todo.completed },
+    )}
+    key={todo.id}
+  >
+    <label className="todo__status-label">
+      <input
+        data-cy="TodoStatus"
+        type="checkbox"
+        className="todo__status"
+        defaultChecked
```

Comment:
```suggestion
        checked={todo.completed}
        onChange={handleCheck}
```

Reply:
implement `handleChange` function

Reply:
Taras, pls be adv that there's no such task in the task list.
TODOS task is divided on three parts.
It's feature should be implemented in the next one😊

Just for info, pls see task list: (https://github.com/dmytro-lebedchenko/react_todo-app-loading-todos)